USAGE_DATA.md

```diff
@@ -0,0 +1,48 @@
+# Data Collection
+
+Podman Desktop uses telemetry to collect anonymous usage data in order to identify issues and improve our user experience. You can read our privacy statement
+[here](https://developers.redhat.com/article/tool-data-collection).
+
+Users are prompted during first startup to accept or decline telemetry. This setting can be
+changed at any time in Settings > Preferences > Telemetry.
+
+On disk this is stored in the `"telemetry.*"` keys within the settings file,
```

Comment:
Maybe replace 'this' by setting as it's not where we store telemetry data but only the setting

Reply:
Fixed in new commit.